Cargo.toml

```diff
@@ -37,6 +54,8 @@ crossbeam-channel = "0.5"
 getopts = "0.2.17"
 surfman = { version = "0.9", features = ["chains", "sm-angle", "sm-angle-default", "sm-x11", "sm-raw-window-handle"] }
 winit = { version = "0.29", features = ["rwh_05"] }
+# Cargo Packager
+cargo-packager-resource-resolver = { version = "0.1", features = ["auto-detect-format"], optional = true }
```

Comment:
I updated a [commit ](https://github.com/european-browser/verso/commit/84825f17246016e64785dbd5744206b64f99004a)that can embed all resource files into binary. I believe we don't need to handle file path then.